prow/plugins/trigger/trigger.go

```diff
@@ -249,9 +228,12 @@ func RunOrSkipRequested(c Client, pr *github.PullRequest, requestedJobs []config
 	toRun := sets.NewString()
 	toSkip := sets.NewString()
 	for _, job := range requestedJobs {
-		runs, err := shouldRun(getChanges, job, pr.Base.Ref, forceRunContexts, body)
-		if err != nil {
-			return err
+		runs := forceRunContexts.Has(job.Context)
```

Comment:
This isn't technically the same as in the past if you added a forced context that didn't run against the branch it would get ignored. Should be fine?